leasing/serializers/land_use_agreement.py

```diff
@@ -431,11 +442,31 @@ class LandUseAgreementUpdateSerializer(
     conditions = LandUseAgreementConditionCreateUpdateSerializer(
         many=True, required=False, allow_null=True
     )
+    plots = LandUseAgreementPlotCreateUpdateSerializer(
+        many=True, required=False, allow_null=True
+    )
 
     class Meta:
         model = LandUseAgreement
         fields = "__all__"
 
+    def update(self, instance, validated_data):
+        if "plots" in validated_data:
+            plots = validated_data.pop("plots")
+            plot_ids = []
+            for plot_item in plots:
+                plot = Plot.objects.get(id=plot_item["id"])
```

Comment:
Should this be wrapped with a `try` and `except` block, just in case there's an exception?

Reply:
It should fail into this if the frontend sents incorrectly JSON data.